deploy/kubernetes/fluentd-sumologic.yaml

```diff
@@ -378,11 +378,51 @@ data:
     <source>
       @type events
     </source>
+    <source>
+      @type prometheus
```

Comment:
do we need to add gem install like https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/83/files#diff-aa72c1ddeec59a31f30834d956165473R29

Reply:
Ah, we already are including it ... we may want to bump the version to 1.4, but never mind this comment.